modules/thunder_gqls/src/Wrappers/EntityListResponse.php

```diff
@@ -51,8 +87,12 @@ public function items() {
       return [];
     }
 
-    $buffer = \Drupal::service('graphql.buffer.entity');
-    $callback = $buffer->add($this->query->getEntityTypeId(), array_values($result));
+    if (empty($this->buffer)) {
+      // phpcs:ignore
+      $this->buffer = \Drupal::service('graphql.buffer.entity');
+    }
+
```

Comment:
```suggestion
```
If we make the suggested change to the constructor then this can be like this. The advantage is that when you remove the deprecation in the constructor you only have to make changes there.